pset-functions.js

```diff
@@ -91,6 +110,30 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+/*function wordCount(phrase) {
+    const wordArray = phrase.split(' ');
+    const count = 0;
+    for (var i = 0; i < wordArray.length; i++) {
+        if (wordArray[i] === phrase) {
+            count++;
+        }
+    }
+    return(wordArray + ':' + count)
+}
+
+console.log(wordCount('olly olly in come free'))*/
+function wordCount(phrase){
+  const wordsExplained = phrase.split(' ').forEach(function(word){
+  const numberofTimes = phrase.match(new RegExp(word, 'g')).length;
+  console.log(`${word}: ${numberofTimes}`)
+})
```

Comment:
@cbrooks2019 be careful, with your indentation here...this line should be indented 1 level to the right (and everything inside the `forEach()` should be indented accordingly as well